python/conv1d_torch_group.py

```diff
@@ -0,0 +1,23 @@
+import json
+import numpy as np
+import torch
+from torch import nn
+
+np.random.seed(1001)
+torch.manual_seed(0)
+
+f = nn.Conv1d(in_channels=6, out_channels=3, kernel_size=1, groups=3, dtype=torch.float64)
```

Comment:
Nice! I was wondering if it might make sense to add a couple more layers, maybe with some different dilations, kernel sizes, etc, just to see if there's any edge cases that we'd need to handle differently.